src/Components/FieldGuide/FieldGuide.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import MushroomCard from '../MushroomCard/MushroomCard';
+import './FieldGuide.css';
+
+const FieldGuide = ({ renderMushroomCards }) => {
+  return (
+    <div className="field-guide-container">
+      <div className="field-guide-content">
+        {renderMushroomCards()}
+      </div>
+    </div>
```

Comment:
I'm happy sharing the renderMushroomCard function cleaned this up a lot!

Reply:
Great refactor for this! Great insight into refactoring our component architecture to do so!

Reply:
I'm going to research best practice so we can change the location of the renderMushroomCards function into a new location/folder if need be... then we will just import the function into the files they are being used in instead of passing it down as prop.